esp-backtrace/src/lib.rs

```diff
@@ -60,7 +60,9 @@ impl BacktraceFrame {
     }
 }
 
+#[allow(unused)]
```

Comment:
Could be `cfg`'d for panic handler, but meh...

Reply:
We have a couple of `allow`s - I will look into reducing them